src/api/issues.rs

```diff
@@ -341,6 +341,29 @@ impl<'octo> IssueHandler<'octo> {
             .await
     }
 
+    /// Replaces all labels for an issue.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// let labels = octocrab::instance()
+    ///     .issues("owner", "repo")
+    ///     .replace_labels(101, &[String::from("help wanted")])
+    ///     .await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub async fn replace_labels(&self, number: u64, labels: &[String]) -> Result<Vec<models::Label>> {
```

Comment:
Minor nit to match the octokit convention. https://octokit.github.io/rest.js/v17#issues-replace-all-labels
```suggestion
    pub async fn replace_all_labels(&self, number: u64, labels: &[String]) -> Result<Vec<models::Label>> {
```